File: Java/reactor-chat/src/main/java/jp/glory/reactor/chat/web/request/LeaveRequest.java
package jp.glory.reactor.chat.web.request;

/**
 * 退室リクエスト.
 * @author gloryof
 *
 */
public class LeaveRequest {

    /**
     * 名前.
     */
    private String name;

    /**
     * @return the name
     */
    public String getName() {
        return name;
    }

    /**
     * @param name the name to set
     */
    public void setName(String name) {
        this.name = name;
    }
}

File: Neo4j/neo4j-study/src/main/kotlin/jp/glory/neo4jstudy/web/request/LeaveRequest.kt
package jp.glory.neo4jstudy.web.request

import java.time.LocalDate

/**
 * 所属から外れるリクエスト.
 *
 * @param leaveAt 退任日
 * @param postId 部署ID
 * @param employeeId 従業員ID
 */
data class LeaveRequest(
    var leaveAt: LocalDate = LocalDate.MAX,
    var postId: Long = 0,
    var employeeId: Long = 0)